app/javascript/react/components/Map/Map.tsx

```diff
@@ -93,18 +117,26 @@ const Map = () => {
       const east = bounds.getNorthEast().lng();
       const west = bounds.getSouthWest().lng();
       setMapBounds({ north, south, east, west });
-      setCurrentZoom(map.getZoom() || DEFAULT_ZOOM);
     },
     [mapInstance]
   );
 
-  const handleMarkerClick = (streamId: React.SetStateAction<number | null>) => {
+  // Handlers
```

Comment:
Does it add value to us? In one line below we can see the useCallback call

Reply:
Same as: https://github.com/HabitatMap/AirCasting/pull/660#discussion_r1633213995